game_event/templates/game-events.html

```diff
@@ -49,6 +52,14 @@ <h1>Game Events</h1>
 				<td>{{ event.min_number_of_players }}</td>
 				<td>{{ event.max_number_of_players }}</td>
 				<td><a href="http://127.0.0.1:8000/courts/{{ event.court.courtID }}">{{ event.court.courtID }}</a></td>
+                <td>{{ event.court.city }}</td>
```

Comment:
same here

Reply:
@SchSeba again in this case, looks file on my IDE : 
<img width="643" alt="image" src="https://github.com/redhat-beyond/MeetBalls/assets/95130756/ed112fef-b252-429d-b32a-e6dc12badc27">

Reply:
@SchSeba Chaned the path. Fixed.

Reply:
weird that is fine then..